benchmark/benchmark_umf.hpp

```diff
@@ -199,7 +199,7 @@ struct disjoint_pool : public pool_interface<Provider> {
             return {nullptr, [](void *) {}};
         }
 
-        ret = umfDisjointPoolParamsSetMinBucketSize(raw_params, 4096);
+        ret = umfDisjointPoolParamsSetMinBucketSize(raw_params, 8);
```

Comment:
why?

Reply:
So we are using more then 1 bucket in benchmarks